ykrt/src/compile/jitc_yk/codegen/x86_64/mod.rs

```diff
@@ -753,7 +754,40 @@ impl<'a> Assemble<'a> {
     }
 
     fn cg_loadtraceinput(&mut self, iidx: jit_ir::InstIdx, inst: &jit_ir::LoadTraceInputInst) {
-        // Find the argument register containing the pointer to the live variables struct.
+        let m = match &self.m.tilocs()[usize::try_from(inst.locidx()).unwrap()] {
+            yksmp::Location::Register(3, ..) => {
```

Comment:
Hey look! This is just (a subset of) the DWARF SysV register mapping!

I'm now about 99% percent sure `yksmp` is using dwarf register numbers. Maybe (later) we can add a comment in `VarLocation::Register` saying so. Then these are no longer magic numbers.

Reply:
It would be good to do that, preferably someone who isn't me, and actually understands this :p

Reply:
I think we need to ask Lukas when he's back. Perhaps we could raise an issue and assign him, as this is probably important enough that we don't want it to get lost?

Reply:
Hopefully that will shake out naturally from #1334.